src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -75,7 +75,10 @@ public PersistentDataStore() {
         String passwordHash = (String) entity.getProperty("password_hash");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         Boolean isAdmin = (Boolean) entity.getProperty("isAdmin");
+        String bio = (String) entity.getProperty("bio");
+        bio = bio == null ? "" : bio;
```

Comment:
Some users will not have the bio field set on our application because they registered before this PR; this should take care of defaulting these users' bios to an empty string upon app starup